addons/test_mail/tests/test_mail_message.py

```diff
@@ -70,19 +70,21 @@ def test_empty_message(self):
         self.assertFalse(message.sudo()._filter_empty(), 'Still having attachments')
 
         # Subtype content
-        note_subtype.write({'description': 'Very important discussions'})
-        record._message_update_content(message, '', [])
-        self.assertFalse(message.attachment_ids)
-        self.assertEqual(message.notified_partner_ids, self.partner_admin)
-        self.assertEqual(message.starred_partner_ids, self.partner_admin)
-        self.assertFalse(message.sudo()._filter_empty(), 'Subtype with description')
+        with self.assertRaises(UserError, msg='Employee does not have the rights to perform on such operation'):
```

Comment:
Purpose of the test was actually not to test access rights, but the result of updating content tool method. Here the write should be done as sudo() to keep the test as it is.